app/src/main/java/net/gsantner/opoc/format/todotxt/SttTask.java

```diff
@@ -10,137 +10,343 @@
 #########################################################*/
 package net.gsantner.opoc.format.todotxt;
 
-import net.gsantner.opoc.preference.MapPropertyBackend;
+import android.widget.TextView;
```

Comment:
All Stt* functionality has been simplified and moved into `SttTask`. `SttCommander` etc are no longer used. The only purpose of the new SttTask is to parse todo.txt lines and extract projects, contexts etc. All modifications are done directly with `RegexReplace` in `TodoTxtTextActions.java`